enigma/src/main/java/org/quiltmc/enigma/impl/analysis/BuiltinPlugin.java

```diff
@@ -54,52 +55,79 @@ private void registerEnumNamingService(EnigmaPluginContext ctx) {
 		final Map<Entry<?>, String> names = new HashMap<>();
 		final EnumFieldNameFindingVisitor visitor = new EnumFieldNameFindingVisitor(names);
 
-		ctx.registerService("enigma:enum_initializer_indexer", JarIndexerService.TYPE, ctx1 -> JarIndexerService.fromVisitor(visitor));
+		ctx.registerService(JarIndexerService.TYPE, ctx1 -> new JarIndexerService() {
+			@Override
+			public void acceptJar(Set<String> scope, ClassProvider classProvider, JarIndex jarIndex) {
+				JarIndexerService.fromVisitor(visitor, "enigma:enum_field_name_finding_visitor").acceptJar(scope, classProvider, jarIndex);
+			}
+
+			@Override
+			public String getId() {
+				return "enigma:enum_initializer_indexer";
+			}
+		});
```

Comment:
Why isn't this just `ctx1 -> JarIndexerService.fromVisitor(visitor, "enigma:enum_initializer_indexer")`?